src/Jeffijoe.MessageFormat/Formatting/Formatters/PluralFormatter.cs

```diff
@@ -139,7 +139,10 @@ internal string Pluralize(string locale, ParsedArguments arguments, double n, do
             Pluralizer pluralizer;
             if (this.Pluralizers.TryGetValue(locale, out pluralizer) == false)
             {
-                pluralizer = this.Pluralizers["en"];
+                if(PluralRulesMetadata.TryGetRuleByLocale(locale, out pluralizer) == false)
```

Comment:
This is absolutely open to discussion, currently, the metadata is used whenever we can't find a user-defined locale with such a name

Reply:
I think that makes sense. I would imagine eventually we remove support for custom pluralizers once we support all of the official locale rules.